tendrl/commons/flows/create_cluster/utils.py

```diff
@@ -88,6 +88,12 @@ def install_python_gdeploy():
             "Failed to install python-gdeploy"
         )
 
+    # Write the python-gdeploy configuration file
+    config_data = "[python-gdeploy]\n" + \
+        "glusterfs_repo = https://buildlogs.centos.org/centos/7/storage/x86_64/gluster-3.9/"
```

Comment:
This is WIP PR and ideally this would configurable (may be in node-agent.conf.yaml).
@nnDarshan in case of rhel gluster repos are configured, would it still need to set this configuration in file?